package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "atomizer",
-  "version": "1.0.1",
+  "version": "2.0.0",
```

Comment:
I could also just publish an alpha or beta release, so we can more easily play around with it as we develop new features.